src/defaultTo.js

```diff
@@ -19,7 +19,8 @@ var _curry2 = require('./internal/_curry2');
  *      defaultTo42(null);  //=> 42
  *      defaultTo42(undefined);  //=> 42
  *      defaultTo42('Ramda');  //=> 'Ramda'
+ *      defaultTo42(parseInt('string')); //=> 42
  */
 module.exports = _curry2(function defaultTo(d, v) {
-  return v == null ? d : v;
+  return v == null || v !== v ? d : v;
```

Comment:
`isNaN` is broken (`isNaN({})` for instance is `true`) and `Number.isNaN` lacks browser support. `NaN` is the only value that isn't equal to itself so this is AFAIK the simplest way to check for it.